capabilities.json

```diff
@@ -82,13 +83,15 @@
               { "value": "Set3", "displayName": "Set3" }
             ]
           },
-          "displayName": "Colorbrewer"
+          "displayName": "Colorbrewer",
+          "displayNameKey": "Visual_FormattingGeneralColorbrewer"
```

Comment:
It'd be better to use `Visual_Formatting_General_Colorbrewer` instead.
Please apply the same format to others